### PR TITLE
bump gl-native to v10.11.0-rc.1, common to v23.3.0-rc.1

### PR DESCRIPTION
**Pull request by alexander-kulikovskii**
### Summary of changes

TBA

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [ ] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

**Review by yunikkk — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,9 +3,17 @@
...
+* Improve performance for style switch use cases by avoiding unneeded tiles re-layout. ([1953](https://github.com/mapbox/mapbox-maps-android/pull/1953))
+
 ## Bug fixes 🐞
 * Fix a bug where `flyTo` animation request invalid tiles from map engine. ([1949](https://github.com/mapbox/mapbox-maps-android/pull/1949))
 * Deprecate `pattern` and `dash` transition properties for layer (e.g. `BackgroundLayer.backgroundPatternTransition`, `FillExtrusionLayer.fillExtrusionPatternTransition`, `FillLayer.fillPatternTransition`, `LineLayer.lineDasharrayTransition`, `LineLayer.linePatternTransition`, ...).  ([1941](https://github.com/mapbox/mapbox-maps-android/pull/1941))
+* Fix terrain tiles missing issue when running in the emulator and some android devices. ([1953](https://github.com/mapbox/mapbox-maps-android/pull/1953))
+
+## Dependencies
+* Update gl-native to v10.11.0-rc.1 and common to v23.3.0-rc.1. ([1953](https://github.com/mapbox/mapbox-maps-android/pull/1953))
+
```
Comment (ank27) [inline CHANGELOG.md:16]:
```suggestion
## Known issues
* If last location is not available, location component will throw a `NullPointerException`. 
```suggestion

```
```

Comment (ank27) [reply CHANGELOG.md:16]: or something of that sort :)

Comment (alexander-kulikovskii) [reply CHANGELOG.md:16]: done

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,9 +3,19 @@
...
+## Known issues
+* If last location is not available, location component will throw a `NullPointerException`.
```
Comment (pengdev) [inline CHANGELOG.md:18]:
```suggestion
* If last location is not available, `LocationEngine` will throw a `NullPointerException`.
```

Comment (alexander-kulikovskii) [reply CHANGELOG.md]: done

**Review by ank27 — APPROVED** [review]

**Pull request merged**